docs/user/exercises/modeling.rst

```diff
@@ -4,6 +4,16 @@ Modeling exercise
     :local:
     :depth: 3
 
+.. note::
+   - Artemis uses a npm library called `Apollon`_ as its modeling editor.
```

Comment:
```suggestion
   - Artemis uses an npm library called `Apollon`_ as its modeling editor.
```
I am not 100% sure but a quick Google search shows me `an npm` is more common than `a npm`. Also, `npm package` might be a more suitable term than `npm library` but I think you can use the one which sounds more correct to you.